assets.php

```diff
@@ -68,21 +63,43 @@ function get_asset_dependencies( string $asset ) : array {
  *
  * @return string The asset's hash.
  */
```

Comment:
This also doesn't have a @param comment. Needed?

Reply:
This has a param tag - it's a line above this.